design/Pipelines.md

```diff
@@ -73,9 +75,11 @@ RenderPipeline CreateRenderPipeline(Device device, const RenderPipelineDescripto
 ```
 
 Translation to the backing APIs would be the following:
- - **D3D12**: Translates to ```ID3D12::CreateComputePipelineState```
+ - **D3D12**: Translates to ```ID3D12::CreateGraphicsPipelineState```. ```IBStripCutValue``` will always be set.
  - **Metal**: Translates to ```MTLDevice::makeRenderPipelineState```
- - **Vulkan**: Translates to ```vkCreateGraphicsPipelines```
+ - **Vulkan**: Translates to ```vkCreateGraphicsPipelines```. ```VkPipelineInputAssemblyStateCreateInfo```'s ```primitiveRestartEnable``` is always set to true. All dynamic states are set on all pipelines.
+
+Open question: should the type of the indices be set in ```RenderPipelineDescriptor```? If not, how is the D3D12 ```IBStripCutValue``` chosen?
```

Comment:
Good question. We don't want to be delaying the PSO creation ever, so my vote is to specify the index type in `RenderPipelineDescriptor` so that D3D12 can set the proper `IBStripCutValue`